src/utilities/keymanager.cc

```diff
@@ -53,7 +54,7 @@ void KeyManager::loadKeys(const std::string *pkey_content, const std::string *ce
     }
     if (!ca.empty()) {
       if (tmp_ca_file == nullptr) {
-        tmp_ca_file = std::make_unique<TemporaryFile>("tls-ca");
+        tmp_ca_file = std_::make_unique<TemporaryFile>("tls-ca");
```

Comment:
This is weird. I get that adding stuff to the `std` namespace is asking for trouble, but in this case, the point is that we won't need to do it anymore after we move to C++14, and we want the switch to be seamless. In this case, it will be anything but, since we will have to change everything back from `std_` to `std`.

Reply:
We could just use "replace all" std_:: to std:: Or You suggest to replace it back now, and disbale the check?

Reply:
I know, replace all is fine, but it's just tedious that we have to change all the code twice. Easier to just let it be, right? I can kind of see it both ways, though, since it is confusing to be adding custom functions to the std namespace. I don't have a strict opinion here.